pkg/nftables/firewall.go

```diff
@@ -149,11 +150,11 @@ func (f *Firewall) validate(file string) error {
 	return nil
 }
 
-func (f *Firewall) reload() error {
-	c := exec.Command(systemctlBin, "reload", nftablesService)
+func (f *Firewall) reload(file string) error {
+	c := exec.Command(nftBin, "-f", file)
```

Comment:
I'm not convinced to use the nft binary directly. `systemctl reload nftables` does more: reads `/etc/nftables.conf` that flushes all rules and loads ipv4 and ipv6 rules. Also I encountered that without the flush over time there is lots of garbage rules and duplicates....

Reply:
OK, thanks will revert

Reply:
done